### PR TITLE
Nudge users to add BP or Blood Sugar after creating patient - 1

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.clubhouse.io/simpledotorg/story/4387/nudge-users-to-add-bp-or-blood-sugar-after-creating-patient

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt [hunk 1]
```diff
@@ -210,12 +210,18 @@ class PatientSummaryEffectHandler @AssistedInject constructor(
...
           .map { loadDataForBackClick ->
             val patientUuid = loadDataForBackClick.patientUuid
+            val countOfRecordedBloodPressures = bloodPressureRepository.bloodPressureCountImmediate(patientUuid)
+            val countOfRecordedBloodSugars = bloodSugarRepository.bloodSugarCountImmediate(patientUuid)
             val medicalHistory = medicalHistoryRepository.historyForPatientOrDefaultImmediate(
                 defaultHistoryUuid = uuidGenerator.v4(),
                 patientUuid = patientUuid
             )
 
-            DataForDoneClickLoaded(countOfRecordedMeasurements = countOfRecordedMeasurements(patientUuid), diagnosisRecorded = medicalHistory.diagnosisRecorded)
```
Comment (janhavisinghh) [inline app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt:218]: Is there a reason why we have not deleted the unused method `PatientSummaryEffectHandler#countOfRecordedMeasurements` yet?

**Review by janhavisinghh — CHANGES_REQUESTED** [review]
LGTM. Left one comment PTAL

File: app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt (hunk 1 above)
Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt:218]: Ah, good catch. Will remove it in the next PR.

Comment (janhavisinghh) [reply app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt:218]: Also, there are some unused parameters in `PatientSummaryEffectHandler`. We can probably remove them as well.

Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt:218]: Yep, they will all be removed in the next PR.

**Review by janhavisinghh — APPROVED** [review]

**Pull request merged**